mozilla/foundation.mozilla.org
# removed blue outline from creepometer when active on safari

Closes #7874
Related PRs/issues #

Link to test page: https://foundation-s-creepomete-virg9i.mofostaging.net/en/privacynotincluded/real-body-piece-better-notice/

**Steps to test**:

1. Please visit the test link above using safari/chrome
2. Scroll down to the creepometer and click on the face/draggable bar
3. Note that there is no longer a blue outline surrounding the area like before
4. If everything looks as expected, testing is complete!
